.github/workflows/ci-maven.yaml

```diff
@@ -87,5 +101,5 @@ jobs:
           cache: maven
       - run: java -version
       - name: Java Linter
-        run: mvn fmt:check
-        working-directory: gapic-generator-java
\ No newline at end of file
+        # Exclude the root project
+        run: mvn -B -ntp --projects '!.' fmt:check
```

Comment:
With this PR, gapic-generator-java alone does not work. So running the fmt:check plugin from the root.